File: day04/animal.py
class Animal:
    def __init__(self):
        self.cnt_hair = 80000
    def eatWell(self):
        self.cnt_hair += 100


if __name__ == '__main__':
    ani = Animal()
    print("cnt_hair", ani.cnt_hair)
    ani.eatWell()
    print("cnt_hair", ani.cnt_hair)

File: day15node/animal.js
class Animal{
    constructor() {
        this.cnt_hair = 80000;
    }


    eatWell() {
        this.cnt_hair += 100;
    }
}

module.exports = Animal;

if (require.main === module) {

    var ani = new Animal();

    console.log("", ani.cnt_hair);
    ani.eatWell();
    console.log("", ani.cnt_hair);
}


